documentation/openapi.yaml

```diff
@@ -99,19 +158,11 @@ paths:
                   k8s_token:
                     type: string
                     example: 'default'
-                  k8s_apiserver:
-                    type: string
-                    example: 'default'
-                  k8s_clustername:
-                    type: string
-                    example: 'default'
-
-
     patch:
       summary: Modify user details
       parameters:
         - in: path
-          name: uid
+          name: id
```

Comment:
what do you think about returning the new user object if the patch was successful? I could set the state without the api-response but a response is more explicit about what happened. That would be helpful for debugging and it increases transparency.

Reply:
Very nice, I like it.